products/index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>COMP</title>
+    <link rel="stylesheet" href="../shared.css" />
+    <link rel="stylesheet" href="./main.css" />
+    <link
```

Comment:
maybe font awesome import should be top of css files.

import font awesome css.
shared css can overwrite font awesome.
main css can overwrite shared css and font awesome css.

Reply:
i don't know the next step when pull request needs fix.
should i reject pull request to get fix?
or you can just fix content of pull request?

Reply:
Thank you for your response. When I do "pull request" next time, I'll include this change about "awesome css".